modules/host/storageobligations.go

```diff
@@ -574,8 +574,15 @@ func (h *Host) removeStorageObligation(so storageObligation, sos storageObligati
 		}
 	}
 	if sos == obligationSucceeded {
+		// Empty obligations don't submit a storage proof. The revenue for an empty
+		// storage obligation should equal the contract cost of the obligation
+		if len(so.SectorRoots) == 0 {
+			h.log.Printf("Not submitted a storage proof for empty contract. Revenue is %v.\n", so.ContractCost.Add(so.PotentialStorageRevenue).Add(so.PotentialDownloadRevenue).Add(so.PotentialUploadRevenue))
```

Comment:
I'd probably print `No need to submit storage proof for empty contract` to make it clear that this is expected behavior.